docs/integrations/containers-orchestration/kubernetes.md

```diff
@@ -363,10 +364,11 @@ Use this dashboard to:
 ### Security Rules Triggered
 
 :::note
-This dashboard relies on Falco. If the Dashboard is not populated, enable Falco by setting the flag `falco:enabled` as `"true"` in values.yaml, as described [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/installation.md).
+This dashboard relies on Falco. If the Dashboard is not populated, enable Falco by setting the flag `falco.enabled` as `"true"` in values.yaml, for details please see
+the Sumo Logic Kubernetes Collection Helm Chart [documentation](https://github.com/SumoLogic/sumologic-kubernetes-collection#documentation).
```

Comment:
Link directly to the Falco section here?

Reply:
Is there any?

Reply:
I don't see any section that we can link here.